rosa_tutorials/rosa-mobb-azure-devops-managed-ocp.adoc

```diff
@@ -0,0 +1,365 @@
+:_content-type: ASSEMBLY
+[id="rosa-mobb-azure-devops-managed-ocp"]
+= Tutorial: Using Azure DevOps with Managed OpenShift
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-azure-devops-managed-ocp
+
+toc::[]
+
+// ---
+// date: '2023-03-14'
+// title: Azure DevOps with Managed OpenShift
+// tags: ["DevOps", "ADO", "Azure", "ARO", "ROSA"]
+// authors:
+//   - Kevin Collins
+// ---
+
+Adopted from link:https://cloud.redhat.com/blog/hosting-an-azure-pipelines-build-agent-in-openshift[Hosting an Azure Pipelines Build Agent in OpenShift]
```

Comment:
Adapted?

Reply:
Good catch, I'm wondering if that line should even be there in the first place haha, but in the meantime I'll make this fix